samples/server/petstore/undertow/src/main/java/org/openapitools/model/Pet.java

```diff
@@ -21,11 +21,16 @@
 @javax.annotation.Generated(value = "org.openapitools.codegen.languages.JavaUndertowServerCodegen")
 public class Pet   {
   
-  private Long id = null;
+  private Long id;
+  
   private Category category = null;
-  private String name = null;
+  
+  private String name;
+  
   private List<String> photoUrls = new ArrayList<String>();
-  private List<Tag> tags = new ArrayList<Tag>();
+
+  private List<Tag> tags = null;
```

Comment:
@ccozzolino thanks for fixing the regression. Should this be `new ArrayList<Tag>()` instead of null?